prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -228,12 +249,63 @@ function generateAndExportLetsEncryptCert() {
         --dns-google-propagation-seconds=600 \
         -d "*.${DOMAIN}"
 
-    TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
+       TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
-}
+    sleep 2
+	#encrypt the private cert
+    encryptCerts
+
+	#copy the cert to cloud
+	gsutil cp "./letsencrypt/live/${DOMAIN}/nightly-gke-tls-integration-app-client-cert.encrypted gs://kyma-prow-secrets/"
+    #copy the private key to cloud
+    gsutil cp "./letsencrypt/live/${DOMAIN}/nightly-gke-tls-integration-app-client-key.encrypted gs://kyma-prow-secrets/"
+
+    gsutil setmeta  -h "Cache-Control:public, max-age=60" gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-key.encrypted
+    gsutil setmeta  -h "Cache-Control:public, max-age=60" gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-client.encrypted
 
+
+}
+function decryptCerts(){
+   local DOMAIN="Nightly"
+  local KYMA_KEYRING="kyma-prow"
+  local KYMA_ENCRYPTION_KEY="projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption"
+  printf "nightly-gke-tls-integration-app-client-key.encrypted"
+  local KYMA_KEYRING="kyma-prow"
+  local KYMA_ENCRYPTION_KEY="projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption"
+    gcloud kms decrypt --location global \
+	--keyring "${KYMA_KEYRING}" \
+	--key "${KYMA_ENCRYPTION_KEY}" \
+	--ciphertext-file letsencrypt/live/"${DOMAIN}"/nightly-gke-tls-integration-app-client-key.encrypted \
+  --plaintext-file letsencrypt/live/"${DOMAIN}"/privkey.pem 
+
+printf "decrypting nightly-gke-tls-integration-app-client-cert.encrypted"
+   gcloud kms decrypt --location global \
+	--keyring "${KYMA_KEYRING}" \
+	--key "${KYMA_ENCRYPTION_KEY}" \
+	--ciphertext-file letsencrypt/live/"${DOMAIN}"/nightly-gke-tls-integration-app-client-cert.encrypted \
+  --plaintext-file "./letsencrypt/live/${DOMAIN}/fullchain.pem"
+}
+function encryptCerts(){
+   local DOMAIN="Nightly"
```

Comment:
It shouldn't be hardcoded

Reply:
variables are removed, they were added to the config.yaml previously